### DISCUSSION TITLE
There is not noscript tag in npmjs website

### DISCUSSION THREAD

**Discussion by saidbakr** [npm/feedback#1081] [Website]
Hello,
Your website, [*npmjs.com*](https://www.npmjs.com), does not has `<noscript>`tag!
It may be regarded as obsoleted behavior, but in some circumstances, it may be useful in the case of disabled JavaScript browsers for any reason.

**Comment by saidbakr** [comment]
As a suggestion:
```html
<noscript>
    <div style="background-color:red;width:100%;margin:-10px;vertical-align:middle;padding:9px">
      <h1>JavaScript is disabled in your browser! Please enable it.</h1>
  </div>
</noscript>
````

**Comment by ljharb** [comment]
Why would JS be required for the functionality of the page? noscript is only used as an undesirable fallback when there’s no graceful degradation possible.

**Reply by saidbakr** [reply]
Do you mean that npmjs.com works fine without JavaScript?!

**Reply by ljharb** [reply]
Most of it does, sure, like most websites. Surely you've tried it without JS, otherwise why would you care about the noscript tag?
